specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -3517,6 +3517,460 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/restorePointCollections/{restorePointCollectionName}": {
+      "put": {
+        "tags": [
+          "RestorePointCollections"
+        ],
+        "operationId": "RestorePointCollections_CreateOrUpdate",
+        "description": "The operation to create or update the restore point collection.",
```

Comment:
Need to provide an aka.ms link to some article on docs.microsoft.com that provides more details and context about Restore Points, pls. work with me Vaibhav on creating the link.